index.md

```diff
@@ -11,6 +11,9 @@ pitch: The OWASP Zed Attack Proxy (ZAP) is one of the world’s most popular fre
 ---
 <!-- rebuild 40 -->
 
+[![OWASP Flagship](https://img.shields.io/badge/owasp-flagship%20project-48A646.svg)](https://www.owasp.org/index.php/OWASP_Project_Inventory#tab=Flagship_Projects)
```

Comment:
The links does not seem to be working properly. Should it link to the new site `/projects#flagships` instead? https://www2.owasp.org/projects#flagships

Reply:
I'll switch it up, unfortunately that link is 100% either but I think that's more of a functional issue. I'll ask about linking to tabs on the new site(s) in slack.